src/main/webapp/style.css

```diff
@@ -65,20 +92,20 @@ body {
   padding-top: 20px;
 }
 
-.sidenav a {
+#sidebar a {
```

Comment:
do we still need this? if not, we can remove it. if so, then we might need a better variable name for it

Reply:
Removed :)